code/__DEFINES/colors.dm

```diff
@@ -71,9 +71,9 @@
 #define COLOR_TEAL "#008080"
 #define COLOR_BLUE "#0000FF"
 #define COLOR_STRONG_BLUE "#1919c8"
-#define COLOR_CENTCOM_BLUE "#134975"
+#define COLOR_CENTCOM_BLUE "#449455" // bubber edit - this is green now - OLD: #134975
 #define COLOR_BRIGHT_BLUE "#2CB2E8"
-#define COLOR_COMMAND_BLUE "#1B67A5"
+#define COLOR_COMMAND_BLUE "#E8BB61" // bubber edit - this is gold now - OLD: #1B67A5
```

Comment:
Not this way, no. I do not want to look at code, see "COLOR_COMMAND_**BLUE**" and have that be actually... gold. It makes code unreadable.